src/admin/shared/components/Sidebar/Sidebar.tsx

```diff
@@ -1,9 +1,10 @@
 import classnames from 'classnames';
-import React, { FunctionComponent } from 'react';
+import * as H from 'history';
+import React, { FunctionComponent, ReactElement } from 'react';
 import { Link, NavLink } from 'react-router-dom';
-
 import { NavigationItemInfo } from '../../../../shared/types';
 
+import { flatten } from 'lodash-es';
```

Comment:
Import order.

Reply:
fixed in: https://github.com/viaacode/avo2-client/commit/705402c33cc42a0e630aa80488a44d3c4eba9439